.github/workflows/pr.yml

```
@@ -6,10 +6,11 @@ on:
 
 jobs:
   unit-tests:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         node-version: [10.x, 12.x]
+        os: [ubuntu-latest, windows-latest]
```

```suggestion
        # os: [ubuntu-latest, windows-latest]
        os: [ubuntu-latest]
```

@Weakky Thinking about how we want to approach this. Rather than bringing windows support as one massive PR can we ship in pieces? But merging e.g. this would mean going red on CI for a long time. My idea to solve that is above, let's comment out the windows part, bringing it back in PRs that make progress, then comment it out again before PR merging.

---

@Akxe That sounds good